src/tests/functional/plugin/shared/include/behavior/ov_infer_request/io_tensor.hpp

```diff
@@ -59,7 +59,6 @@ struct OVInferRequestCheckTensorPrecision : public testing::WithParamInterface<O
 
     std::vector<ov::element::Type> precisions = {
```

Comment:
Should be moved to optional, not removed

Reply:
@sbalandi ?

Reply:
fixed